### DISCUSSION TITLE
My current daily driver rice of Labwc

### DISCUSSION THREAD

**Discussion by Speyll** [labwc/labwc#2440] [Show and tell]
![Screenshot](https://i.ibb.co/NnkDQ0K/grim.webp)

I wanted to share my current rice for this amazing compositor and express my gratitude to everyone involved in its development. Stacking WMs or compositors are quite rare, so having a Wayland one this stable and polished is truly a joy. You’re doing incredible work!  

I’ve been using Labwc extensively for about 9 months now, and it’s been a fantastic experience especially with the addition of tablet pencil pressure support. I used to switch to Sway for drawing, but now I can do everything on Labwc, and it’s been an absolute treat.

## Software Used

  - `foot` Terminal emulator.
  - `waybar` Status bar.
  - `fuzzel` The application launcher menu.
  - `pipewire` Audio server.
  - `grim` For taking screenshots.
  - `slurp` To select regions for screenshot capture. 
  - `brightnessctl` Tool to control screen brightness.
  - `wlsunset` For managing night mode settings.
  - `wl-clipboard` For clipboard management.
  - `cliphist` To access clipboard history.
  - `swayimg` My image viewer.
  - `fnott` Notification system.
  - `mpv` Video player.
  - `nvim` Text editor
  - `noto-fonts-emoji`, `noto-fonts-ttf`, `noto-fonts-cjk`,`nerd-fonts-symbols-ttf`, `cascadia-mono` Fonts.
  - `flavours` For setting base16 colors pretty much everywhere.

**Comment by johanmalm** [comment]
Looks great. Thanks for sharing :smile:

**Comment by MTSYuki** [comment]
What System Monitor you using on the left at the bottom?

**Reply by MTSYuki** [reply]
NVM, I founded: [gotop](https://github.com/xxxserxxx/gotop)
